src/services/headerbidding.js

```diff
@@ -13,6 +14,7 @@ export function initializeBiddingServices({
   amazon = false
 }) {
   if (window.arcBiddingReady) {
+    sendLog('Arcbidding is already ready; return without initializing bidding services again.');
```

Comment:
This log can be updated to `fetchBids(): Header bidding has been previously initialized`